packages/tables/resources/views/columns/text-column.blade.php

```diff
@@ -42,6 +42,13 @@
         },
         'whitespace-normal' => $canWrap(),
     ]) }}
+    
+    @if ($isCopyable())
```

Comment:
I think you should be able to have an icon AND a copy button at once. Or, at least, I can imagine that we will get that request within a week 😂

Reply:
I changed it to be set as default when activating, however, it can be customized if you want in a customized way.

Reply:
Yeah but people should be able to use an icon & the copy button. At the moment, they cant use an icon too if they use the copy button

Reply:
I added an extra function that adds the copy icon and position at the end, for those who want a copy icon pattern.